app/tests/test_assets_endpoint.py

```diff
@@ -308,8 +308,8 @@ def test_asset_endpoint_post_characters_geoadmin_variant(self):
         self.assertEqual(
             {
                 'geoadmin:variant': [
-                    'Invalid geoadmin:variant, special characters beside one '
-                    'space are not allowed',
+                    'Invalid geoadmin:variant "more than twenty-five characters with s", '
```

Comment:
what does with s" mean?

Reply:
between the "" is the value of the geoadmin:variant, so `more than twenty-five characters with s` is the value in the samples why it has an s at the end its a mistery :-)